package.json

```diff
@@ -26,7 +26,7 @@
     "atom": ">=1.0.0 <2.0.0"
   },
   "dependencies": {
-    "atom-languageclient": "^0.6.1",
+    "atom-languageclient": "^0.7.0",
```

Comment:
I had another PR that updated the deps that I finally merged. Mind just pulling master into your branch?